package.json

```diff
@@ -23,7 +23,7 @@
     "@contentstack/core": "^1.1.0",
     "@contentstack/utils": "^1.3.8",
     "@types/humps": "^2.0.6",
-    "axios": "^1.7.4",
+    "axios": "^1.7.3",
```

Comment:
@nadeem-cs Can you axios v1.7.3 have ssr vulnerability issue, Please update to axios v1.7.4

Reply:
@sunil-lakshman that can be added to your PR. This is just a revert PR.

Reply:
Okay